.env.example

```diff
@@ -1,5 +1,7 @@
-VUE_APP_AF_API_KEY=''
+VUE_APP_API_KEY=''
 VUE_APP_BASE_URL_PLIO=''
 VUE_APP_AWS_SQS_ACCESS_KEY=''
-VUE_APP_AWS_SQS_SECRET_ACCESS_KEY=''
+VUE_APP_AWS_SQS_SECRET_KEY=''
 VUE_APP_BASE_URL_MEET=''
+VUE_APP_AWS_REGION=''
+VUE_APP_AWS_SQS_URL=''
```

Comment:
Looks good!
Can we do some organizing here? Like a comment like "# aws" and all the things related to that go under that comment, and similarly for others. For example, right now we have two env variables here for redirection targets, so maybe we can add that section as well (just as a comment)
What do you think?

Reply:
I'm not sure I understand this comment. The comments on env variables are in the docs.

Reply:
Yeah I explained it pretty bad 🤦 
We'll discuss on call